store/src/lora_valid_poc.rs

```diff
@@ -98,9 +108,14 @@ impl TryFrom<LoraValidBeaconReport> for LoraValidBeaconReportV1 {
 impl TryFrom<LoraValidWitnessReportV1> for LoraValidWitnessReport {
     type Error = Error;
     fn try_from(v: LoraValidWitnessReportV1) -> Result<Self> {
+        let location = v
+            .location
+            .is_empty()
+            .not()
+            .then(|| v.location.parse::<u64>().unwrap());
```

Comment:
nope.. don't unwrap.. you have a Result as an error.. also I don't think you need to do is_empty.not at all here

Reply:
kinda have to unwrap here, otherwise i get

mismatched types
expected enum `std::option::Option<u64>`
   found enum `std::option::Option<std::result::Result<u64, std::num::ParseIntError>>`

Reply:
call .transpose()